cfme/storage/manager.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+
+from navmazing import NavigateToAttribute
+from widgetastic.utils import Version, VersionPick
+from widgetastic.widget import View, NoSuchElementException, Text
+from widgetastic_manageiq import (
+    Accordion,
+    BootstrapSelect,
+    BootstrapTreeview,
+    BreadCrumb,
+    ManageIQTree,
+    PaginationPane,
+    SummaryTable,
+    Table
+)
+from widgetastic_patternfly import (
+    Button,
+    Dropdown,
+    FlashMessages
+)
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from cfme.utils.appliance import BaseCollection, BaseEntity
+
+
+class ManagerToolbar(View):
+    """The toolbar on the Storage Manager or Provider page"""
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+
+
+class ManagerDetailsToolbar(View):
+    """The toolbar on the Storage Manager or Provider detail page"""
+    reload = Button(title='Reload Current Display')
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    monitoring = Dropdown('Monitoring')
+    download = Button(title='Download summary in PDF format')
```

Comment:
QUESTION: are you sure it always has such title ? 
Besides that, are you sure this is Button not Dropdown ?

Reply:
yes, it's not a dropdown, a simple button with pdf icon, having title "Download summary in PDF format"